example/package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "build": "webpack --config webpack.config.js"
   },
-  "license": "AGPL-3.0",
+  "license": "LGPL-3.0",
```

Comment:
The `example` source code is Unlicense, so I think that should be the one used here too